packages/fxa-payments-server/src/components/PlanDetails/index.tsx

```diff
@@ -134,119 +215,138 @@ export const PlanDetails = ({
                 ))}
               </ul>
 
-              <div className="plan-details-total">
-                {coupon && coupon.discountAmount && (
+              {loading ? (
+                <div className="loading">
```

Comment:
Replace `loading` with `pb-4`.

This is equivalent to `1rem` (from TW docs):

> pb-4	padding-bottom: 1rem; /* 16px */

Reply:
done